app/editor/src/upload-context/UploadContext.ts

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
Move files associated with upload-context to `/app/editor/src/components/upload`.  It's a shared component that will eventually be moved to `tno-core`.

Reply:
I thought I had it all working but I must have had some cached items. But for some reason I cannot get the context wrapper to behave as expected when I move it to `/app/editor/src/components/upload`. The wrapper all of a sudden acts as a block to the rest of the context providers. Do you have any idea why that may be? Has been driving me mental. Works fine when I leave the folder as is.

Reply:
Shouldn't matter where you put it.  But some times hot reload gets cache mixed up.  `make refresh n=editor`

Reply:
I'm not sure whether this is a local issue or not but I cannot get rid of it - don't want to waste too much more time on it. Makes no sense. I change the same things when dragging it to and from its current location. Works from src no issue...